daprdocs/content/en/reference/api/workflow_api.md

```diff
@@ -90,6 +90,112 @@ Traceparent: 00-e3dedffedbeb9efbde9fbed3f8e2d8-5f38960d43d24e98-01
 Connection: close 
 ```
 
+## Pause workflow request
+
+Pause a running workflow instance with the given name and instance ID.
+
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/<workflowComponentName>/<instanceId>/pause
+```
+
+### URL parameters
+
+Parameter | Description
+--------- | -----------
+`workflowComponentName` | Current default is `dapr` for Dapr Workflows
+`instanceId` | Unique value created for each run of a specific workflow
+
+### HTTP response codes
+
+Code | Description
+---- | -----------
+`202`  | Accepted
+`400`  | Request was malformed
+`500`  | Request formatted correctly, error in dapr code or underlying component
+
+### Response content
+
+The API call will provide a response similar to this:
+
+```bash
```

Comment:
@RyanLettieri @cgillum I used the responses for "terminate workflow" as placeholders for pause/resume/purge - can you review and let me know what needs to be changed?